ppd/ppd-test.c

```diff
@@ -141,8 +141,10 @@ cups_array_t *ppdTest(int ignore,          // Which errors to ignore
   ppd = NULL;
   status = ERROR_NONE;
   root = rootdir;
-  output = ppdArrayNew("");
-  ppdArrayAdd(output," ");
+  output = cupsArrayNew3(NULL, NULL, NULL, 0,
```

Comment:
Please add the NULL check for `output` as I did in my suggestion.

Reply:
The null check for output results in NULL being returned everytime testppdfile is run.

Reply:
Try to add the following, right after the `output = cupsArrayNew3(...);` expression:
```
if (output == NULL)
{
  if (log) log(ld, ...);
  return (NULL);
}
```